yt/frontends/amrvac/tests/test_outputs.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+
+import yt
+from yt.utilities.answer_testing.framework import \
+    requires_ds, \
+    data_dir_load, small_patch_amr
+from yt.testing import requires_file
+
+blastwave_cartesian_3D = "amrvac/bw_3d0000.dat"
+blastwave_polar_2D = "amrvac/bw_polar_2D0000.dat"
+blastwave_spherical_2D = "amrvac/bw_2d0000.dat"
+blastwave_cylindrical_3D = "amrvac/bw_cylindrical_3D0000.dat"
+khi_cartesian_2D = "amrvac/kh_2D0000.dat"
+khi_cartesian_3D = "amrvac/kh_3D0000.dat"
+jet_cylindrical_25D = "amrvac/Jet0003.dat"
+riemann_cartesian_175D = "amrvac/R_1d0005.dat"
+
+@requires_ds(blastwave_cartesian_3D)
+def test_domain_size(self):
+    """Check for correct box size, see bw_3d.par"""
+    ds = data_dir_load(blastwave_cartesian_3D)
+    for lb in ds.domain_left_edge:
+        assert int(lb) == 0
+    for rb in ds.domain_right_edge:
+        assert int(rb) == 2
+    for w in ds.domain_width:
+        assert int(w) == 2
+
+@requires_file(blastwave_cartesian_3D)
+def test_grid_attributes(self):
```

Comment:
```suggestion
def test_grid_attributes():
```

Reply:
actually no I never knew it was even *possible* to run the answer tests on my laptop. Will the `@requires_file` decorator download data from the yt-data server ? Is it as simple as running `nosetest` locally ?

Reply:
apparently not... To clarify, I was using pytest to run tests locally before submitting this PR. Since the tests were updated with the `@requires_ds` decorator I've been committing in the blind :-(